RecDP/setup.py

```diff
@@ -75,5 +75,7 @@
         "tabulate==0.9.0",
         "sentencepiece==0.1.99",
         "selectolax",
+        "Faker==19.6.2",
+        "torch==2.0.1",
```

Comment:
PDT may be very specific to torch version, any specific reason we need to fix it to 2.0.1

Reply:
change to 1.13.1